File: 1-fundamentos/bloco4-introducao-a-javascript-e-logica-de-programacao/3/practice.js
let fruits = [3, 4, 10, 1, 12];

let sum = 0;

for (let index = 0; index < fruits.length; index += 1) {
    sum = sum + fruits[index];
}

if (sum > 15) {
    console.log(sum);
} else {
    console.log("valor menor que 16");
}



const number = 100;

let sum = 0;

for (let index = 0; index <= number; index += 1) {
    sum += index;
}

console.log(sum);

File: 4-ciencia-da-computacao/secao3-estruturas-de-dados-I-listas-lineares/2-arrays/practice/practice.py
# 1- Tipos abstratos de dados possuem uma única implementação bem documentada?

# Não.

# 2- Listas(list), dicionários(dict) e conjuntos(set), que já vêm no Python, são considerados tipos de dados? Caso negativo, justifique sua resposta.

# Sim.

# 3- Que tal adicionarmos um método update que atualiza o valor a partir de um índice?
# A assinatura deve ser def update(self, index, value):


class ListArray:
    def __init__(self):
        self.data = []

    def len(self):
        return len(self.data)

    def __str__(self):
        return str(self.data)

    def get(self, index):
        return self.data[index]

    def set(self, index, value):
        return self.data.insert(index, value)

    def remove(self, index):
        return self.data.pop(index)

    def update(self, index, value):
        self.data[index] = value
